COMPILING.md

```diff
@@ -255,6 +255,10 @@ require manual modification of build files.
    to generate IDE projects by supplying the `-G` flag.  Run `cmake -G` for a
    comprehensive list of supported back-ends.
 
+   Note that if you do not specify the `build` directory the current directory
+   will be the build target. This will modify many version controlled files
+   and be messy to fix.
```

Comment:
Rather than adding documentation I'd prefer one of the options discussed in https://stackoverflow.com/questions/1208681/with-cmake-how-would-you-disable-in-source-builds/10306476#10306476 to disallow this behaviour.

Reply:
@tautschnig I added https://github.com/diffblue/cbmc/pull/5782 for this.

Reply:
Removed as this is now superseded by PR #5782.